CustomCraftSML/Serialization/Entries/AliasRecipe.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace CustomCraft2SML.Serialization.Entries
+{
+    using System.Collections.Generic;
+    using Common.EasyMarkup;
+    using CustomCraft2SML.Interfaces;
+
+    internal class AliasRecipe : AddedRecipe, IAliasRecipe
+    {
+        internal new static readonly string[] TutorialText = new[]
+        {
+            "AliasRecipe: A powerful tool with multiple applications.",
+            "    Alias recipes allow you to create multiple ways to craft the same item,",
+            "      bypassing one of the limitations of Subnautica's crafting system.",
+            "    Alias recipes can also be used to add your own custom items into the game, all without any coding, with some limitations.",
+            "    Alias recipes have all the same properties of Added Recipes, but when creating your own items, you will want to include these new properties:",
+            "        DisplayName: Sets the in-game name for the new item.",
+            "        Tooltip: Sets the in-game tooltip text whenever you hover over the item.",
+            "        FunctionalID: Choose an existing item in the game and clone that item's in-game functions into your custom item." +
+            "            Without this property, any user created item will be non-functional in-game, usable as a crafting component but otherwise useful for nothing else.",
+        };
+
+        private readonly EmProperty<string> displayName;
+        private readonly EmProperty<string> tooltip;
+        private readonly EmProperty<string> functionalID;
```

Comment:
`AliasRecipes` could also have an optional `SpriteItemID` property.  
That would allow users to have it use a preexisting icon from an item that doesn't appear within the list of linked items.